### DISCUSSION TITLE
Is there a possibility for requirements management like Sphinx-Needs is offering

### DISCUSSION THREAD

**Discussion by PeterStange** [docToolchain/docToolchain#1443] [General]
In our company more and more teams switched to Sphinx because of Sphinx-Needs and its possibility to manage requirements, specifications inside the doc-as-code document itself. 
I'm wondering if there is a possibility with asciidoc to achieve the same or similar. 
Does anybody knows more about this feature support in asciidoc, or can give hints whom to contact? 
Maybe, someone already did it using docToolchain tasks.
For me, it sounds like a good extension as I see the acceptance of Sphinx-Needs in the communities.
Any comment is appreciated.

**Comment by rdmueller** [comment]
Hi Peter, 
it seems that different industries have different needs. I never had to work with a tool like Sphinx needs.
But i am sure that we could achieve the same results with AsciiDoc and docToolchain.
The Challenge is to find someone who knows both worlds in order to write some requirements and maybe extend docToolchain. But with good requirements, there is also the chance that someone else could extend docToolchain.

What do you think?

**Reply by PeterStange** [reply]
Hello Ralf
Yes, I could imagine that a similar solution is feasible with asciidoc and docToolchain. I hoped, that there is already somebody aware of it and could give hints how to solve/realize it. 
I by myself, didn't used Sphinx Needs until now. Even Sphinx itself is something I'm not used to. 
But, if we want to offer docToolchain for future tasks, we should have a closer look to it to understand the 'Needs'.

**Reply by TorstenC** [reply]
In meiner Diskussion mit ChatGPT über die Integration von Sphinx Needs in eine AsciiDoc-Umgebung kam die Frage auf, ob eine verlustfreie bidirektionale Konvertierung zwischen AsciiDoc (.adoc) und reStructuredText (reST) möglich ist. 

ChatGPT hat dazu angemerkt, dass Sphinx stark auf reST fokussiert ist und dass die Sphinx Funktionen, Erweiterungen und Optionen standardmäßig reST voraussetzen. Das macht es schwierig, AsciiDoc direkt zu integrieren. Eine verlustfreie bidirektionale Konvertierung zwischen .adoc und reST wäre zudem ein komplexes Unterfangen aufgrund der unterschiedlichen syntaktischen und semantischen Strukturen beider Formate.
